packages/react-composites/src/composites/CallComposite/components/CallReadinessModal.tsx

```diff
@@ -61,6 +68,8 @@ export const CallReadinessModal = (props: {
           return (
             <CameraAndMicrophoneDomainPermissions
               appName={'app'}
+              /* @conditional-compile-remove(unsupported-browser) */
```

Comment:
What happens if CallReadiness goes before unsupported browser to GA? will this break the app? these modals need the browser type right? or does it just default to the non safari strings if this isn't release yet?

Reply:
It uses the useragent when unsupported browser is not available